closed/make/CreateJars.gmk

```diff
@@ -60,6 +66,7 @@ $(eval $(call SetupArchive,BUILD_TRACEFORMAT_JAR, , \
 	SKIP_METAINF := true))
 
 JARS += $(patsubst %,$(IMAGES_OUTPUTDIR)/lib/ext/%.jar, dtfj dtfjview traceformat)
+JARS += $(patsubst %,$(IMAGES_OUTPUTDIR)/lib/%.jar, diagnostics)
```

Comment:
I don't think there is any need to create another jar file, the content can go into the tools.jar

Reply:
Testing that now...

Reply:
Tested and working.